### DISCUSSION TITLE
2020/05/cara-membuat-sitemap-otomatis-di-wordpress

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#25] [General]
# 2020/05/cara-membuat-sitemap-otomatis-di-wordpress

Dalam artikel yang akan saya buat kali ini akan membahas tips untuk sobat pengguna wordpress, dari pada saya lupa lebih baik saya buatkan saja dalam bentuk tutorials. sebelum lebih dalam menjelaskan ada kalanya mengenal terlebih dahulu tentang sitemap ini.

https://www.mantankode.net/2020/05/cara-membuat-sitemap-otomatis-di-wordpress.html

**Comment by madi10** [comment]
**Tebal**

_Miring_

(Atasi Rontok Rambut)[https://tokopedia.link/UJzCFyoIwsb]

![Atasi Rontok Rambut](https://raw.githubusercontent.com/madi10/MANTAN/master/img/7.jpg.webp)

```<?php $paged=(get_query_var('paged'))?get_query_var('paged'):1;$args=array('posts_per_page'=>4000,'post_type'=>'post','orderby'=>'title','order'=>'ASC','paged'=>$paged);query_posts($args);if(have_posts()){$in_this_row=0;while(have_posts()){the_post();$first_letter=strtoupper(substr(apply_filters('the_title',$post->post_title),0,1));```
